INSTALLATION.md

```diff
@@ -1,26 +1,28 @@
-# Installing the Justice40 Tool
+# Justice40 Tool Installation
 
 *[¡Lea esto en español!](INSTALLATION-es.md)*
```

Comment:
Are you up for taking a pass at updating the ES version of this file too? Otherwise maybe @esfoobar-usds might be able to?

Reply:
Unfortunately I don't think I'd be qualified to do that, my Spanish would definitely not look right to a native speaker.

Reply:
No worries at all!